docs/section21/Rule21-18.md

```diff
@@ -0,0 +1,57 @@
+
+# Boiler - Rule 21-18  
+
+**Rule ID:** 21-18  
+**Rule Description:** Baseline shall have only one heating hot water plant.  
+**Rule Assertion:** B-RMR LEN(FluidLoop.type == "HEATING") == 1  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B-RMR  
+**Applicability Checks:**  
+
+1. P-RMR is not modeled with purchased heating.
```

Comment:
I think this rule should apply to projects with purchased heating as well.

Reply:
as discussed RDS will be updated to include PHW, b systems and c systems will be added to the applicability check